web/l10n/en.json

```diff
@@ -340,6 +340,7 @@
     "alarmTampering": "Tampering Alarm",
     "alarmRemoving": "Removing Alarm",
     "notificationType": "Type of Notification",
+    "notificationAlways": "Send Always",
```

Comment:
I think we should rename label because it's a bit confusing. Am I understanding correctly that this flag means that notification works for all user devices? If yes, we should probably use "All Devices" or something like this for the label. Internally we can still call it "always", unless we can think of better name.